### ISSUE TITLE
msi x399 gaming pro carbon ac

### ISSUE THREAD

**Issue by mrdev023** [nagisa/msi-rgb#40]
But i don't know the ship used 😥

**Comment by nagisa** [comment]
From other reports it is very likely that colour is changing, but is only changing for one of the "lightning areas", like an LED under a RAM or something.

**Comment by ChairmanMeow** [comment]
What is the solution here? I am running into a similar issue here

**Comment by dcerisano** [comment]
`msi-rgb` does not initialise the rgb header (JLED1). You need to boot to windows, run the MSI Gaming app and turn on the LEDs. This has to be done after every cold boot (from unplugged).    

If you carefully watch windows boot at startup thereafter you will notice the MSI Gaming app resets the LEDs to red, then to the color you chose every time.

**Comment by ChairmanMeow** [comment]
@dcerisano so is the solution to send over the init codes?

**Comment by dcerisano** [comment]
Yes, if you can track them down please post them here are even submit a pull request.
You would have to sniff the port registers during the MSI Gaming App start up sequence, I think.

**Comment by ShayBox** [comment]
I also have the `x399 GAMING PRO CARBON AC`, can I help out to get this working?

**Comment by Advantaged** [comment]
Any news or further development in the topic?
On this motherboard is/are "Demo-LED" connections, do someone know how to connect or have the specifications?
Note:I haven't Windows at all, my sensors are: NCT7802Y, NCT7904D, NCT1008 but all marked with "No" and under `/etc/module` is listed only an `lm92`.
My goal is to switch all LED to blue or switch off.
